src/EnergyPlus/Boilers.cc

```diff
@@ -451,6 +460,27 @@ void BoilerSpecs::SetupOutputVars(EnergyPlusData &state)
                         "Heating",
                         "Boiler Parasitic",
                         "Plant");
+    if (this->FuelType != Constant::ResourceType::Electricity) {
+        SetupOutputVariable(state,
+                            "Boiler Ancillary " + this->BoilerFuelTypeForOutputVariable + " Rate",
+                            OutputProcessor::Unit::W,
+                            this->ParasiticFuelRate,
```

Comment:
Agreed. For the time being.

Reply:
Times up. I keep thinking about this one. How do we know how the user wants to use this input? Is the user smart enough to know that a contactor draws say 50 W when the electric boiler is active? Maybe, maybe not. But if this report is not available then this input won't be included. I lean towards leaving this open to the user.

Reply:
@mjwitte As you probably know, I am just following the naming convention that was already in place.

Reply:
@rraustad These outputs should never show up for an electric boiler because there are existing output variables right above in the code for "Boiler Ancillary Electricity Rate" and "Boiler Ancillary Electricity Energy", which are for the Parasitic Electric Load field (which is an on-cycle parasitic). So the output variable names would show up twice.